python/mlcroissant/mlcroissant/_src/operation_graph/operations/field_test.py

```diff
@@ -53,3 +54,13 @@ def test_cast_value_image(open_mock):
     expected = field._cast_value(b"PNG...Some image...", DataType.IMAGE_OBJECT)
     open_mock.assert_called_once()
     assert expected == "opened_image"
+
+
+def test_extract_lines():
+    with tempfile.TemporaryDirectory() as tempdir:
+        path = tempdir + "/file.txt"
+        with open(path, "wb") as f:
+            f.write(b"bonjour\nh\xc3\xa9llo\n")
```

Comment:
Is Croissant's "lines" behavior supposed to be the same as Python's [splitlines](https://docs.python.org/3/library/stdtypes.html#str.splitlines)? If yes, should tests cover '\r' and other characters?

Reply:
Done.

I thought `splitlines` was only for str, but it plays nicely with bytes as well. I also updated the unit test (even though now it almost boils down to testing `splitlines` itself). Thanks!